### DISCUSSION TITLE
Chocolatey package support

### DISCUSSION THREAD

**Discussion by flcdrg** [Azure/azure-dev#682] [Show and tell]
I've created a Chocolatey package to facilitate installing/upgrading the Azure Developer CLI

https://community.chocolatey.org/packages/azure-developer-cli

**Comment by danieljurek** [comment]
@flcdrg thanks for building a Chocolatey package! We have that on our backlog. Would you be interested in working with us to make this an official part of our releases?

**Reply by flcdrg** [reply]
Absolutely!

I also happen to be a volunteer package moderator for Chocolatey, and I know we love it when software publishers take over the role of producing the packages.

**Comment by jongio** [comment]
Closed via: https://github.com/Azure/azure-dev/pull/1538
